src/Sylius/Bundle/AdminBundle/Resources/views/Order/Field/state.html.twig

```diff
@@ -0,0 +1,5 @@
+{% import '@SyliusUi/Macro/labels.html.twig' as label %}
+
+{% set text = 'sylius.ui.'~data %}
+
+{{ label.default(text|trans) }}
```

Comment:
I believe that without parentheses it will translate only `data`, not full `sylius.ui.data` string.